l4rm4nd_VoucherVault_myapp_views_59.py

```
@@ -0,0 +1,775 @@
+        return HttpResponse("No file found", status=404)
+
+@require_GET
+@login_required
+def serve_image_file(request, item_id):
+    item = get_object_or_404(Item, id=item_id)
+
+    if not has_item_access(item, request.user):
+        return HttpResponse("Unauthorized", status=403)
+
+    if not item.file:
+        raise Http404("No file attached.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)